Submitting files

RavidDimant opened the pull request:
Uploaded 7 "Harry Potter" books for TF-IDF example in NLP notebook.
Uploaded "Alice in Wonderland" for W2V example in NLP notebook. 
Uploaded 08 MLlib & MLlib_NLP notebooks.

RavidDimant committed:
Add files via upload

Uploading 7 Harry Potter books & Alice book.

RavidDimant committed:
Delete work/08 MLlib.ipynb

delete existing file and replace it

RavidDimant committed:
Add files via upload

Uploading 2 ipynb files.

cnoam commented:
typing it again: The Harry potter books are copyrighted. you MUST NOT put them here (unless you have prior written permission  from owner

cnoam reviewed (commented):
I don't understand. Why did you delete the file?
This is not the way to do it

cnoam reviewed (commented):
The commit name should describe what was done in the commit.
Example:
Added TF-IDF example to notebook
<empty line>
The TF-IDF example show sdf asd asd 
asd asd adsf asd asdf
asd asd asdf
 asdf

RavidDimant committed:
Add files via upload 

Added 6 books by F. Scott Fitzgerald from project Gutenberg for TF-IDF example in MLlib_NLP notebook.

RavidDimant committed:
Delete data/HPBook1.txt

Delete because of copyright

RavidDimant committed:
Delete data/HPBook2.txt

Delete because of copyright

RavidDimant committed:
Delete data/HPBook3.txt

Delete because of copyright

RavidDimant committed:
Delete data/HPBook4.txt

Delete because of copyright

RavidDimant committed:
Delete data/HPBook5.txt

Delete because of copyright

RavidDimant committed:
Delete data/HPBook6.txt

Delete because of copyright

RavidDimant committed:
Delete data/HPBook7.txt

Delete because of copyright

RavidDimant committed:
Delete work/MLlib_NLP.ipynb

Delete old NLP notebook for uploading a new version later, featuring an updated TF-IDF example.

RavidDimant committed:
Add files via upload

Add a new NLP notebook with an updated TF-IDF example.
